tests/test_unit_logger.c

```diff
@@ -25,6 +28,47 @@ void test_log_str_to_level(void **unused) {
 }
 
 #ifndef _WIN32
+/**
```

Comment:
Please add a test case to cover the behavior in log_init(), e.g. having a config file in place, after calling snowflake_global_init() check the log settings are as expected.